R/read_staff_popfeb20.R

```diff
@@ -0,0 +1,25 @@
+#' Get Feb 20, 2020 population data for applicable rows in the fac_data
```

Comment:
Do we use this function anywhere? // I think it's useful, but curious why it ended up here!

Reply:
I wanted to add it in if were to start using staff data denoms anywhere. Not using yet.